common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -485,4 +485,253 @@ aluneth = {
     artifacts = {
 		aluneth = { value = 1 }
     }
+}
+felomelorn = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 42002		# Anastarian
+		}
+	}
+    artifacts = {
+		felomelorn = { value = 1 }
+    }
+}
+hammer_of_the_twilight = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+		modifier = {
+			factor = 0
+			NOT = { year = 585 }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			is_playable = yes
+			
+			culture = twilights_hammer
+		}
+	}
+    artifacts = {
+		hammer_of_the_twilight = { value = 1 }
+    }
+}
+hammer_of_the_lightbringer = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60036						# Uther Lightbringer
+		}
+	}
+    artifacts = {
+		hammer_of_the_lightbringer = { value = 1 }
+    }
+}
+flute_of_accuracy = {
+	max_amount = 5
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			OR = {
+				character = 42086						# Alleria Windrunner
+				character = 42002						# Anasterian Sunstrider
+				character = 42089						# Idorial Windrunner
+			}
+		}
+	}
+    artifacts = {
+		flute_of_accuracy = { value = 1 }
+    }
+}
+lionhorn_of_stormwind = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 1008						# Anduin Lothar
+		}
+			additive_modifier = {
+			value = 100
+			character = 8							# Varian Wrynn
+		}
+	}
+    artifacts = {
+		lionhorn_of_stormwind = { value = 1 }
+    }
+}
```

Comment:
Can spawn in 9 years old Varian inventory

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67306343-7e0b0500-f4ff-11e9-9be4-b4b47908322f.png)

</details>

Reply:
fixed, Varian get's it at start, if Anduin's dead now